client/wfnews-war/src/main/angular/src/app/services/report-of-fire-service.ts

```diff
@@ -117,30 +124,39 @@ export class ReportOfFireService {
       const image3 = rofJson.image3;
 
       const formData = new FormData()
-      if (resource !== null) formData.append('resource', resource)
+      if (resource) formData.append('resource', resource)
+
+      if (image1) formData.append('image1', image1)
+      if (image2) formData.append('image2', image2)
+      if (image3) formData.append('image3', image3)
 
-      if (image1 !== null) formData.append('image1', image1)
-      if (image1 !== null) formData.append('image2', image2)
-      if (image1 !== null) formData.append('image3', image3)
       try {
         // Make an HTTP POST request to your server's API endpoint
         let response = await fetch(rofUrl, {
           method: 'POST',
           body: formData
-      });
+      })
     
-        if (response.ok) {
-          // The server successfully processed the report
-          return { success: true, message: 'Report submitted successfully' };
-        } else {
+      if (response.ok) {
+           // The server successfully processed the report
+         return { success: true, message: 'Report submitted successfully' };
```

Comment:
Probably should remove the data from local storage after success?

Reply:
Thanks, done